lib/main.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+var bluebird, dateService, request, URI;
+
+URI = require("urijs");
+bluebird = require("bluebird");
+dateService = require("./date-service")();
+request = require("request-promise");
+
+/**
+ * Creates the main manager object. The main purpose is to inject dependencies.
+ *
+ * @param {Object} args The args from docopt.
+ * @return {manager}
+ */
+module.exports = (args) => {
+    var logger, manager, newArgs, ShelfFile, ShelfMetadata, utils;
+
+    newArgs = require("./arguments")(args);
+    logger = require("./logger")(newArgs);
+
+    logger.debug(`Raw args: ${JSON.stringify(args, null, 4)}`);
+    logger.debug(`Cleaned args: ${JSON.stringify(newArgs, null, 4)}`);
+
+    logger.debug("Requiring \"./utils\"");
```

Comment:
"Loading" may be a more accurate way of describing what's going on, despite the fact that you're actually running "require".